ipa-core/src/protocol/ipa_prf/malicious_security/verifier.rs

```diff
@@ -73,12 +73,12 @@ fn compute_sum_share<F: PrimeField, const λ: usize, const P: usize>(zkp: &[F; P
 /// This function compresses the `u_or_v` values and returns the next `u_or_v` values.
 ///
 /// The function uses streams since stream offers a chunk method.
-fn recurse_u_or_v<'a, F: PrimeField, J, const λ: usize>(
+fn recurse_u_or_v<'a, F: PrimeField, J, const λ_FIRST: usize, const λ: usize>(
```

Comment:
This feels bad because we only need this on the first call to `recurse_u_or_v`, and yet we call this method many times.